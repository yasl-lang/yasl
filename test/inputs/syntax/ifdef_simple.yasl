ifdef A {
    echo "A = #{A:r}"
}

ifdef B {
    echo "B = #{B:r}"
}

echo args
