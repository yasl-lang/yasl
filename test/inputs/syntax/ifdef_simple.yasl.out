[test/inputs/syntax/ifdef_simple.yasl]
